app/controllers/api/v1/reviews_controller.rb

```diff
@@ -5,18 +5,35 @@ def create
     if user_signed_in?
       # Design descision to be made: will the react element need the wizard?
       if new_review.save
-        render json: { wizard: wizard , review: review_params }
+        render json: { wizard: wizard, review: review_params }
       else
-        render json: { wizard: wizard , review: review_params, errors: new_review.errors.full_messages }, status: :unprocessable_entity
+        render json: { wizard: wizard, review: review_params, errors: new_review.errors.full_messages }, status: :unprocessable_entity
```

Comment:
we would probably just respond with the errors here. Good job on the status code.